cli/valet.php

```diff
@@ -289,6 +289,61 @@
     })->descriptions('Change the version of php used by valet', [
         'phpVersion' => 'The PHP version you want to use, e.g php@7.2',
     ]);
+
+    /**
+     * Tail log files.
+     */
+    $app->command('logs [-f|--follow] [-l|--lines=] [key]', function ($follow, $lines, $key = null) {
+        $defaultLogs = [
+            'php' => VALET_HOME_PATH.'/Log/php.log',
+            'php-fpm' => '/usr/local/var/log/php-fpm.log',
+            'nginx' => VALET_HOME_PATH.'/Log/nginx-error.log',
+            'mysql' => VALET_HOME_PATH.'/Log/mysql.log',
+            'mailhog' => '/usr/local/var/log/mailhog.log',
+            'redis' => '/usr/local/var/log/redis.log',
+        ];
+
+        $configLogs = data_get(Configuration::read(), 'logs');
+        if (!is_array($configLogs)) $configLogs = [];
+
+        $logs = collect(array_merge($defaultLogs, $configLogs))->sortKeys();
+
+        if (!$key) {
+            info(implode(PHP_EOL, [
+                'Here are the logs you might be interested in.',
```

Comment:
Can we move the "Here are the logs" line to just above the table? So make it the last line of text instead of the first?

Reply:
Done! (cb1136e7a616b9c0231e99f95d23beb5686b0a0c)